src/ThemeManager.ts

```diff
@@ -1,19 +1,11 @@
 import * as vscode from "vscode";
-import { DokiSticker, DokiTheme, StickerType } from "./DokiTheme";
-import {
-  InstallStatus,
-  installStickers,
-  installWallPaper,
-  removeStickers,
-} from "./StickerService";
-import { VSCodeGlobals } from "./VSCodeGlobals";
-import { StatusBarComponent } from "./StatusBar";
-import {
-  showStickerInstallationSupportWindow,
-  showStickerRemovalSupportWindow,
-} from "./SupportService";
+import {DokiSticker, DokiTheme, StickerType} from "./DokiTheme";
+import {InstallStatus, installStickers, installWallPaper, removeStickers,} from "./StickerService";
+import {VSCodeGlobals} from "./VSCodeGlobals";
```

Comment:
Ah, this might be an artifact of me switching editors during development. VSCode has prettier installed, and I just use the default for WebStorm. So I think I ran code formatting in WebStorm here.

I tend to ignore whitespace changes in my diffs and imports are normally collapsed :smile: